### ISSUE TITLE
Error received when running gulp command

### ISSUE THREAD

**Issue by brianyang** [google/closure-compiler-js#11]
when running `gulp` I see the following error: 

```
Error: Cannot find module 'gulp-sourcemaps'
```

**Comment by clpacheco90** [comment]
Hi brianyang, 
 didi you try to install the gulp-sourcemaps 
npm install -save gulp-sourcemaps

**Comment by bighappyface** [comment]
I am having a similar issue but with different output:

```shell
$ node_modules/.bin/gulp
/path/to/chromedev/closure-compiler-js/samples/node_modules/google-closure-compiler-js/lib/gulp.js:116
  return function(compilationOptions={}, pluginOptions={}) {
                                    ^

SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/path/to/chromedev/closure-compiler-js/samples/node_modules/google-closure-compiler-js/index.js:25:9)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/path/to/chromedev/closure-compiler-js/samples/gulpfile.js:21:18)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Liftoff.handleArguments (/path/to/chromedev/closure-compiler-js/samples/node_modules/gulp/bin/gulp.js:116:3)
    at Liftoff.<anonymous> (/path/to/chromedev/closure-compiler-js/samples/node_modules/liftoff/index.js:198:16)
    at module.exports (/path/to/chromedev/closure-compiler-js/samples/node_modules/flagged-respawn/index.js:17:3)
    at Liftoff.<anonymous> (/path/to/chromedev/closure-compiler-js/samples/node_modules/liftoff/index.js:190:9)
    at /path/to/chromedev/closure-compiler-js/samples/node_modules/liftoff/index.js:164:9
    at /path/to/chromedev/closure-compiler-js/samples/node_modules/v8flags/index.js:108:14
    at /path/to/chromedev/closure-compiler-js/samples/node_modules/v8flags/index.js:36:12
    at /path/to/chromedev/closure-compiler-js/samples/node_modules/v8flags/index.js:47:7
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickCallback (node.js:349:13)
    at Function.Module.runMain (module.js:443:11)
    at startup (node.js:139:18)
    at node.js:974:3
```

This happens on node v4.5.0, v5.12.0, and v6.5.0.

It's probably me, but I figured this may be helpful for others if I/we can get it figured out.

**Comment by bighappyface** [comment]
#16 perhaps?

**Comment by clpacheco90** [comment]
Hi I've got the same issue. You need to do a pull.
https://github.com/google/closure-compiler-js/issues/13

**Comment by samthor** [comment]
These should be fixed as of [20160828.0.1](https://github.com/google/closure-compiler-js/releases/tag/20160828.0.1), now available on NPM.
